packages/storybook/src/installation.stories.mdx

```diff
@@ -22,9 +22,9 @@ It is recommended to install the package [Tailwindcss-animate](https://github.co
 
 Execute the following command in your terminal to install Mochi UI:
 
-```sh
+<Source code={`
 pnpm add @mochi-ui/core @mochi-ui/theme tailwindcss-animate
-```
+`} />
```

Comment:
Fixing bug on newest version of storybook

https://github.com/storybookjs/storybook/issues/24729

![image](https://github.com/consolelabs/mochi-ui/assets/71743905/d723eedc-bd74-4625-88b1-4aa651b0c1c2)